### ISSUE TITLE
Deprecation warning for "buf beta push"

### ISSUE THREAD

**Issue by charithe** [bufbuild/buf-push-action#4]
Y'all are probably already aware of this:
 
```
Run bufbuild/buf-push-action@v0.2.0
Command "push" is deprecated, "buf beta push" has been moved to "buf push".
We recommend migrating, however this command continues to work.
```
 
It's not a big deal. Just thought I'd point it out. Happy to create a PR myself to change the command to `buf push` if it helps.

**Comment by charithe** [comment]
Additionally, it looks like `branch` flag has been replaced by the `tag` flag in the latest buf releases. The action still uses the former.

**Comment by johanbrandhorst** [comment]
Ah, that's definitely wrong - branches have been deprecated to simplify the user experience. I think you mean the `branch` parameter is expected, looking at https://github.com/bufbuild/buf-push-action/blob/main/src/run.ts#L58-L63?

**Comment by amckinney** [comment]
This was resolved in [`v0.3.0`](https://github.com/bufbuild/buf-push-action/releases/tag/v0.3.0). Thanks for the issue!
